### PR TITLE
383 blog css

### LINKED ISSUE #383: Adjust CSS for blog posts
1.  Please make links visibly different (color, highlighting, whatever) so that they stand out as clickable - right now they seem exactly the same font, color, and background as plain text.

2.  Please add a bit of space between numbered list items - they are very crowded right now (see the latest post on statistics functions for examples).

3.  Please narrow the layout so that the text of blog posts is the same width as the text of guide pages (i.e., there is a gutter on both sides rather than just on the left). Right now, the lines are too long horizontally for comfortable reading.

And speaking of guide pages, is there a reason to have the navigation menu for block categories on the right rather than on the left?

### PR DESCRIPTION
**Pull request by MayaGans**
Updates:

- Adding line height so not just list elements but all text has a little more breathing space 
- Reformatting the blog landing page to make links more obvious
- Left align sticky guide TOC

Closes #383

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -9,9 +9,10 @@
 
 <div class="posts">
   {% for post in site.posts %}
-  <article>
-    <h2>{{ post.date | date: "%Y-%m-%d" }}: <a href="{{ post.url }}">{{ post.title }}</a></h2>
-    {{ post.excerpt }}
+  <article style="padding-bottom:40px;margin-right:15rem;">
```
Comment (gvwilson) [inline blog.html]: Can these inline `style` elements be made part of the CSS?

File: blog.html [hunk 2]
```diff
@@ -9,9 +9,10 @@
...
+    <p style="color:grey;">{{ post.date | date: "%Y-%m-%d" }}</p>
```
Comment (gvwilson) [inline blog.html]: Please make inline style part of the CSS.

File: blog.html [hunk 3]
```diff
@@ -9,9 +9,10 @@
...
+    <h2><a href="{{ post.url }}">{{ post.title }}</a></h2>
+    <div style="color:grey;">{{ post.excerpt }}</p>
```
Comment (gvwilson) [inline blog.html]: Part of the CSS, please make this inline style.

**Review by gvwilson — CHANGES_REQUESTED** [review]
Please move a few inline style elements to the CSS and then go ahead and merge.

**Comment by MayaGans** [comment]
Ah I'm having trouble having the scss file auto render with `jekyll serve -I` so I test/play with css directly in the file - sorry about not cleaning up, should be good to go now

**Pull request merged**